ci/rpm/test_daos.sh

```diff
@@ -7,7 +7,7 @@ set -uex
 if git show -s --format=%B | grep "^Skip-test: true"; then
   exit 0
 fi
```

Comment:
This should be handled using `when` in `Jenkinsfile`.
```suggestion
fi
```

Reply:
Will be in the next push